### DISCUSSION TITLE
anybody make a webui yet?

### DISCUSSION THREAD

**Discussion by gruevy** [suno-ai/bark#276] [General]
I'm not big on doing everything manually from the command line, so I haven't really bothered with Bark yet. I'd like to, though, since I have some books that might be fun to narrate. Does anyone have anything they've written for it they could share?

**Comment by JonathanFly** [comment]
https://github.com/JonathanFly/bark

As long as you can stand a billion emojis...

Should be a big late tonight or tomorrow with an installer. You can also try 'git checkout dev' for some extra stuff that somewhat works.

**Reply by gruevy** [reply]
Awesome, thanks. I'll check it out.
